docker-compose.yml

```diff
@@ -1,30 +0,0 @@
-version: '2'
```

Comment:
I like to have a complete development environment contained within the repository.  Hence this file.  I shall re-instate